### PR TITLE
change Readme

### PR DESCRIPTION
**Pull request by shelbychen97**
This change is [Reviewable](https://reviewable.io/reviews/m-lab/annotation-service/29)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,49 @@
...
+    - Latitude
+    - Longitude
+  2. Location File
+    - GeonameID
+    - Continent Code
+    - Country ISO
+    - Country Name
+    - City Name
+    - Metro Code
+
```
Comment (yachang) [inline README.md]: geoname_id is the same with registered_country_geoname_id most of time, but with some exceptions. 
Either geoname_id or registered_country_geoname_id could be not available for some IP addresses.

**Comment by yachang** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[README.md, line 9 at r3](https://reviewable.io:443/reviews/m-lab/annotation-service/29#-KsiE1fQ7Hn3aunasE7w:-KsiE1fQ7Hn3aunasE7x:b-fbeh8q) ([raw file](https://github.com/m-lab/annotation-service/blob/dde4bdaf771b04905794c865009e34407b874eb0/README.md#L9)):*
> ```Markdown
> provided below.
> 
> MaxMind GeoLiteLatest databases include:
> ```

What is the filename for the 1 and 2?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/annotation-service/29)*

**Comment by yachang** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[README.md, line 23 at r3](https://reviewable.io:443/reviews/m-lab/annotation-service/29#-KsiE6PLTqS-nChtfoMd:-KsiE6PLTqS-nChtfoMe:bjo4tcr) ([raw file](https://github.com/m-lab/annotation-service/blob/dde4bdaf771b04905794c865009e34407b874eb0/README.md#L23)):*
> ```Markdown
>     - MetroCode
> 
> MaxMind GeoLite2 databases include:
> ```

filenames too

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/annotation-service/29)*

**Comment by yachang** [comment]
:lgtm:

---

Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/annotation-service/29#-:-KsiOKZGnORrsw2e2Aek:bnfp4nl)*

**Comment by gfr10598** [comment]
Review status: 0 of 1 files reviewed at latest revision, 5 unresolved discussions.

---

*[README.md, line 3 at r4](https://reviewable.io:443/reviews/m-lab/annotation-service/29#-KsiUhIJCze57_kA9PXt:-KsiUhIJCze57_kA9PXu:bo81tby) ([raw file](https://github.com/m-lab/annotation-service/blob/99beabc6e597aff0d8cf532735aa39a852f3d31c/README.md#L3)):*
> ```Markdown
> Annotation integration service provides geolocation for IPv4 and IPv6 MaxMind databases from Google Cloud Storage.
> 
> If an annotation request is dated prior to August 2017, location data will be derived from
> ```

I would suggest adding something like:

In (date), MaxMind added the GeoLite2 data format, which is richer and more efficient than the earlier GeoLite format.  
Prior to August 2017, mlab only collected the original GeoLite snapshots, so if an annotation request is dated ... 

---

*[README.md, line 4 at r4](https://reviewable.io:443/reviews/m-lab/annotation-service/29#-KsiTe5mw-Fpa2gD-PiO:-KsiTe5mw-Fpa2gD-PiP:blp12j3) ([raw file](https://github.com/m-lab/annotation-service/blob/99beabc6e597aff0d8cf532735aa39a852f3d31c/README.md#L4)):*
> ```Markdown
> 
> If an annotation request is dated prior to August 2017, location data will be derived from
> MaxMind GeoLiteLatest databases. Otherwise, data will be provided by
> ```

Drop the Latest suffix.  Although web search finds the right info, MaxMind doesn't use the Latest suffix, so it is quite confusing.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/annotation-service/29)*

**Pull request merged**